Bitte auch die Linter-Fehler beheben, sonst baut es nicht. Siehe Build: https://github.com/BreakOutEvent/breakout-frontend/runs/7018338460?check_suite_focus=true

```
/home/runner/work/breakout-frontend/breakout-frontend/src/client/js/profile.js
  40:7   error  Strings must use singlequote                    quotes
  41:28  error  Strings must use singlequote                    quotes
  48:19  error  Strings must use singlequote                    quotes
  55:13  error  Expected indentation of 14 spaces but found 12  indent
  55:25  error  Strings must use singlequote                    quotes
  56:13  error  Expected indentation of 14 spaces but found 12  indent
  58:32  error  Strings must use singlequote                    quotes
  62:11  error  Expected indentation of 8 spaces but found 10   indent
  62:17  error  Strings must use singlequote                    quotes
  62:98  error  Missing semicolon                               semi
  ```

--- a/src/client/js/profile.js
+++ b/src/client/js/profile.js
@@ -59,7 +59,7 @@ $(document).ready(() => {
               alert('Es gab ein Problem bei der Löschung Ihres Kontos.');
             }
             return res;
-          })
+          });
       }
       else {
         return false;

--- a/src/client/js/profile.js
+++ b/src/client/js/profile.js
@@ -60,7 +60,6 @@ $(document).ready(() => {
             }
             return res;
           })
-          .then(res => console.log(res));
       }
       else {
         return false;

--- a/src/client/js/profile.js
+++ b/src/client/js/profile.js
@@ -34,32 +34,33 @@ $(document).ready(() => {
     if (!confirmNewPasswordValue) return;
     const matches = $('#newPassword').val() === confirmNewPasswordValue;
     $('#confirmNewPasswordContainer')[matches ? 'removeClass' : 'addClass']('has-error');
-  }); 
+  });
 
   $(function () {
-    $("#delete_user").click(function () {
-      var result = confirm("Sind Sie sicher, dass Sie Ihr Konto löschen möchten?\nDann werden ihre personenbezogenen Daten gelöscht und ihre Beiträge auf der Website anonymisiert.");
+    $('#delete_user').click(function () {
+      var result = confirm('Sind Sie sicher, dass Sie Ihr Konto löschen möchten?\nDann werden Ihre personenbezogenen Daten gelöscht und Ihre Beiträge auf der Website anonymisiert.');
       const accessToken = window.boUserData.access_token;
       const user = window.boUserData.me.id;
       const apiUrl = window.boClientConfig.baseUrl;
 
       if (result == true) {
         fetch(`${apiUrl}/user/${user}/user/`, {
-          method: "DELETE",
+          method: 'DELETE',
           headers: {
             authorization: `Bearer ${accessToken}`
           },
         })
           .then(res => {
             if (res.ok) {
-            console.log("successful");
-            return window.location.href = '/logout';
+              alert('Ihr Konto wurde erfolgreich gelöscht!\nIhre Daten werden überall anonymisiert.');
+              return window.location.href = '/logout';
             }
-            else { console.log("unsuccessful"); }
+            else {
+              alert('Es gab ein Problem bei der Löschung Ihres Kontos.');
+            }
             return res;
           })
           .then(res => console.log(res));
-          alert("Ihr Konto wurde erfolgreich gelöscht!\nIhre Daten werden überall anonymisiert.")
       }
       else {
         return false;
@@ -167,7 +168,7 @@ $(document).ready(() => {
 
   });
 
-  
+
 
   $('#teammembers_form').submit(function (e) {
     e.preventDefault();
@@ -176,8 +177,7 @@ $(document).ready(() => {
       var data = new FormData($('#teammembers_form')[0]);
       const result = {};
 
-      for (var entry of data.entries())
-      {
+      for (var entry of data.entries()) {
         result[entry[0]] = entry[1];
       }
       toggleLoading('#teammembers_CTA');

